### PR TITLE
F/cloud 1855

### PR DESCRIPTION
**Pull request by sdoguparti**
# Description
Please refer to the ticket for details

Fixes [#8 ]
https://drivevariant.atlassian.net/browse/CLOUD-1855

## Type of change
Bug Fix

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
This has been tested using the demo projects for python, DotNet, NodeJs. All of the details about the test cases has been mentioned in the comments of the ticket mentioned above

- [x] Sanity Testing

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by genghis-tuan — COMMENTED** [review]
Looks great. Can you please update the PR info to include the ticket, testing steps, and check the checkboxes?

You can also squash the commits.

File: entrypoint.sh [hunk 1]
```diff
@@ -48,4 +48,31 @@ if [[ 1 -gt "$(cat $GITHUB_ENV | grep -c AWS_REGION)" ]]; then
...
     fi
-fi
\ No newline at end of file
+fi
+
+DEFAULT_SONAR_PROJECT_KEY=$(echo "$GITHUB_REPOSITORY" | tr / _ )
+echo "DEFAULT_SONAR_PROJECT_KEY=$DEFAULT_SONAR_PROJECT_KEY"
+
+# Setting the project key and project name for sonar property values coming from dotnet app
+SONAR_PROJECT_KEY_INPUT=$SONARCLOUD_PROJECT_KEY
```
Comment (naveen-variant) [inline entrypoint.sh:57]: Form where would these keys come SONARCLOUD_PROJECT_KEY, SONARCLOUD_PROJECT_NAME

Comment (salyateem13) [reply entrypoint.sh:57]: These are set in the workflows on dotnet projects. 

**ie https://github.com/variant-inc/myvariant-eventhandler/blob/b688afbec32d7ea1d6715e78ed684d83d166b7e2/.github/workflows/branch-sonar.yaml#L11**

Comment (salyateem13) [reply entrypoint.sh:57]: So it defaults to that but get overwritten if sonarproject property files doesn't exist.. which it shouldn't for dotnet

Comment (salyateem13) [reply entrypoint.sh:57]: Can we document this better @sdoguparti. Everything looks good otherwise.

Comment (salyateem13) [reply entrypoint.sh:57]: https://github.com/variant-inc/actions-dotnet. Should create a ticket to update actions-dotnet docs https://github.com/variant-inc/actions-dotnet and in confluence with sonarcloud  setup

**Pull request merged**